cargo-near/src/commands/new/new-project-template/Cargo.toml.template

```diff
@@ -6,11 +6,28 @@ edition = "2021"
 # TODO: Fill out the repository field to help NEAR ecosystem tools to discover your project.
 # NEP-0330 is automatically implemented for all contracts built with https://github.com/near/cargo-near.
 # Link to the repository will be available via `contract_source_metadata` view-function.
-#repository = "https://github.com/xxx/xxx"
+#repository = "https://github.com/<xxx>/<xxx>"
 
 [lib]
 crate-type = ["cdylib", "rlib"]
 
+# fields to configure build with WASM reproducibility, according to specs  
+# in https://github.com/near/NEPs/blob/master/neps/nep-0330.md 
+[package.metadata.near.reproducible_build]
+# docker image, descriptor of build environment
+image = "dj8yfo/sourcescan:0.x.x-dev-git-pull-134" # ERROR: replace with tag in `sourcescan/cargo-near` 
+# tag after colon above serves only descriptive purpose; image is identified by digest
+image_digest = "sha256:526548c0db16aa335c806c509d39ea10d0511cf5d005f556fb38d507f929df3b"
+# build command inside of docker container 
+# if docker image from default gallery is used https://hub.docker.com/r/sourcescan/cargo-near/tags,
+# the command may be any combination of flags of `cargo-near`,
+# supported by respective version of binary inside the container besides `--no-locked` flag
+container_build_command = ["cargo", "near", "build"]
+# TODO: fill out the 
+# git remote, where contract's source code will be cloned from for verification
+# this may often be concatenation of `repository` above and ".git" suffix
+source_code_git_url = "https://github.com/<xxx>/<xxx>.git"
```

Comment:
Can we just use the `repository` field? It is clonnable without .git suffix

Reply:
On an different matter, let's consider a fork scenario, where contributors will not push their commits to the upstream repo while testing, but may want to deploy (and build) smoothly (and ideally with reproducible build, but let's see what would be required for that)